pylink/jlink.py

```diff
@@ -830,6 +830,26 @@ def set_log_file(self, file_path):
         if res:
             raise errors.JLinkException(res)
 
+    @open_required
+    def set_script_file(self, file_path):
+        """Sets a custom JLink Script file to use
+        see https://wiki.segger.com/J-Link_script_files
+
+        Args:
+          self (JLink): the ``JLink`` instance
+          file_path (str): file path to the JLink script file to be used
+
+        Returns:
+          ``None``
+
+        Raises:
+          JLinkException: if the path specified is invalid.
+
+        """
```

Comment:
```suggestion
        """Sets the custom `Script File`_ to use.

        Args:
          self (JLink): the ``JLink`` instance
          file_path (str): file path to the JLink script file to be used

        Returns:
          ``None``

        Raises:
          JLinkException: if the path specified is invalid.

        .. _Script File:
          https://wiki.segger.com/J-Link_script_files
        """
```

Reply:
Minor NIT here, but I think the format for linking in the Google Documents format is this style: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Chose that style because it was previously used in `set_script_file()` and I was going for consistency.
Happy to change.